blue-team/linux/change-passwords.sh

```diff
@@ -17,13 +17,15 @@ if [[ -z "$passTemp" ]] || [[ "$passTemp" != "$passConfirm" ]]; then
 	exit 2
 fi
 
-# host=$(hostname)
-
 for user in $(awk -F: '$7 ~ /(\/sh|\/bash)/ { print $1 }' /etc/passwd); do
 	if [[ $(id -u $user) -eq 0 ]]; then # Don't change password for root
 		continue
 	fi
+  touch ./users.csv
 	echo "Changing password for $user"
 	newPass="$passTemp$delim$user"
 	echo -e "$newPass\\n$newPass" | passwd "$user"
+  echo -e "$user,$newPass\n" > users.csv
```

Comment:
Wouldn't you need to use `>>` to append?

Reply:
Ah yes, nice catch